pywinauto/controls/HwndWrapper.py

```diff
@@ -1278,13 +1278,16 @@ def wheel_mouse(self, distance, pressed = "", coords = (0, 0), count = 1, retry_
             scroll_type = self._scroll_types["up"]["end"]
         else:
             raise ValueError('Distance should be expressed in multiples or divisions of WHEEL_DELTA and should not be equal to zero!')
-        mouse_flag = pressed
-        self.wparam = coeff
+        self.wparam = [distance,pressed]
+        print("self.wparam:")
+        print(self.wparam)
         self.lparam = coords
+        print("self.lparam:")
+        print(self.lparam)
         if retry_interval is None:
             retry_interval = Timings.scroll_step_wait
         while count > 0:
-            self.send_message(win32defines.WM_MOUSEWHEEL, mouse_flag, scroll_type)
+            self.send_message(win32defines.WM_MOUSEWHEEL, win32structures.WPARAM(distance))
```

Comment:
You need to pack high word and low word into wParam using `win32functions.py` functions `HiWord` and `LoWord` and maybe `MakeLong`. Read the MSDN docs very carefully please.